portfolio/src/main/webapp/script.js

```diff
@@ -174,17 +180,37 @@ function submitForm(form) {
     },
     body: JSON.stringify(formData),
   }).then(response => {
-    loadComments();
-    document.getElementById("newcommentform").reset();
+      loadComments();
+      document.getElementById("newcommentform").reset();
   });
 }
 
+/**
+ * Submits a post request to /delete-data to delete all comments,
+ * then reloads the comments section
+ */ 
 function clearComments() {
   fetch("/delete-data", {
     method: 'POST',
     headers: {
-      'Content-Type': 'application/json',
+      'Content-Type' : 'application/json',
     },
     body: ''
   }).then(response => loadComments());
+}
+
+/**
+ * Toggles sort order button between ascending and descending on click
+ * and then reloads comment section to effect the change
+ */
+function changeSortOrder() {
+  const sortOrderButton = document.getElementById("sortorder");
+  if (sortOrderButton.className === "des") {
+    sortOrderButton.className = "asc";
+    sortOrderButton.innerHTML = '&uarr;';
```

Comment:
Optional: As a future PR, you can also consider using little triangle icons instead of the text.

Reply:
To add on that: use http://google.github.io/material-design-icons/#icon-font-for-the-web